Examples/KlaviyoSwiftExamples/Shared/LoginView.swift

```diff
@@ -0,0 +1,111 @@
+                            .font(.title2)
+                            .fontWeight(.semibold)
+                            .foregroundColor(.secondary)
+
+                        VStack(spacing: 20) {
+                            VStack(alignment: .leading, spacing: 8) {
+                                Text("Zipcode")
+                                    .font(.subheadline)
+                                    .fontWeight(.medium)
+                                    .foregroundColor(.primary)
+
+                                TextField("Enter your zipcode", text: $zipcode)
+                                    .textFieldStyle(.roundedBorder)
+                                    .keyboardType(.default)
+                                    .font(.body)
+                            }
+
+                            VStack(alignment: .leading, spacing: 8) {
+                                Text("Email")
+                                    .font(.subheadline)
+                                    .fontWeight(.medium)
+                                    .foregroundColor(.primary)
+
+                                TextField("Enter your email", text: $email)
+                                    .textFieldStyle(.roundedBorder)
+                                    .keyboardType(.emailAddress)
+                                    .autocapitalization(.none)
+                                    .font(.body)
+                            }
+                        }
+
+                        HStack {
+                            Text("Remember Me")
+                                .font(.subheadline)
+                                .foregroundColor(.secondary)
+
+                            Spacer()
+
+                            Toggle("", isOn: $rememberMe)
+                                .toggleStyle(SwitchToggleStyle(tint: .blue))
+                        }
+
+                        Button(action: login) {
+                            Text("Get Started")
+                                .font(.headline)
+                                .fontWeight(.semibold)
+                                .foregroundColor(.white)
+                                .frame(maxWidth: .infinity)
+                                .padding(.vertical, 16)
+                                .background(
+                                    LinearGradient(
+                                        gradient: Gradient(colors: [.blue, .blue.opacity(0.8)]),
+                                        startPoint: .leading,
+                                        endPoint: .trailing
+                                    )
+                                )
+                                .cornerRadius(12)
+                                .shadow(color: .blue.opacity(0.3), radius: 8, x: 0, y: 4)
+                        }
+                        .disabled(email.isEmpty && zipcode.isEmpty)
+                        .opacity(email.isEmpty && zipcode.isEmpty ? 0.6 : 1.0)
```

Comment:
```suggestion
                        .disabled(isFormInvalid)
                        .opacity(isFormInvalid ? 0.6 : 1.0)
```

⬆️ will work if you add ⬇️ 

```swift
private var isFormInvalid: Bool {
    email.isEmpty || zipcode.isEmpty
}
```